contents.json

```diff
@@ -24,19 +26,31 @@
             "name": "Image Classification",
             "description": "Image classification with PyTorch",
             "directory": "pytorch-image-classification",
-            "tags": [ "PyTorch", "timm", "Computer Vision", "Classification" ]
+            "tags": [ "PyTorch", "timm", "Computer Vision", "Classification", "Multi-label"]
         },
         {
             "name": "Instance Segmentation",
             "description": "Instance segmentation using Detectron2",
             "directory": "detectron2-instance-segmentation",
             "tags": [ "detectron2", "Computer Vision", "Segmentation" ]
         },
+        {
+            "name": "Audio Classification",
+            "description": "Audio classification with PyTorch",
+            "directory": "pytorch-audio-classification",
+            "tags": [ "PyTorch", "timm", "Audio", "Classification", "Multi-label"]
+        },
         {
             "name": "Sorghum -100 Cultivar Identification - FGVC 9",
             "description": "Image classification with PyTorch",
             "directory": "pytorch-kaggle-sorghum",
             "tags": [ "PyTorch", "timm", "Computer Vision", "Classification", "Kaggle" ]
-        }
+        },
+        {
+            "name": "BirdCLEF 2022",
+            "description": "Identify bird calls in soundscapes",
+            "directory": "pytorch-kaggle-bird-clef",
+            "tags": [ "PyTorch", "timm", "Audio", "Classification", "Multi-label"]
+        },
```

Comment:
JSON does not allow trailing commas, that is leading to the issues @hornof is running into.